docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -60,10 +62,15 @@ The properties specific to IBFT 2.0 are:
 * `blockperiodseconds` - The minimum block time, in seconds.
 * `epochlength` - The number of blocks after which to reset all votes.
 * `requesttimeoutseconds` - The timeout for each consensus round before a round change, in seconds.
+* `blockreward` - Optional reward amount in Wei to reward the beneficiary. Defaults to zero (0).
+    Can be specified as a hexidecimal (with 0x prefix) or decimal string value.
+* `miningbeneficiary` - Optional beneficiary of the `blockreward`. Defaults to the validator
```

Comment:
@rain-on - are `blockreward` and `miningbeneficiary` network level settings or node level settings?

Reply:
They MUST be network level - if someone in the network has a value different to that of the block proposer, the block import will fail on that node (due to a state-root mismatch, i.e. proposer paid blockReward to address-X, receiving node paid address-Y, meaning the world state at the end of the block is different between the nodes).